src/Command/LockCommand.php

```diff
@@ -37,64 +53,121 @@ protected function configure(): void
         $this
             ->setName('ems:contenttype:lock')
             ->setDescription('Lock a content type')
-            ->addArgument('contentType', InputArgument::REQUIRED, 'content type to recompute')
-            ->addArgument('time', InputArgument::REQUIRED, 'lock until (+1day, +5min, now)')
-            ->addOption('user', null, InputOption::VALUE_REQUIRED, 'lock username', 'EMS_COMMAND')
-            ->addOption('force', null, InputOption::VALUE_NONE, 'do not check for already locked revisions')
-            ->addOption('if-empty', null, InputOption::VALUE_NONE, 'lock if there are no pending locks for the same user')
-            ->addOption('ouuid', null, InputOption::VALUE_OPTIONAL, 'lock a specific ouuid', null)
+            ->addArgument(self::ARGUMENT_CONTENT_TYPE, InputArgument::REQUIRED, 'content type to recompute')
+            ->addArgument(self::ARGUMENT_TIME, InputArgument::REQUIRED, 'lock until (+1day, +5min, now)')
+            ->addOption(self::OPTION_QUERY, null, InputOption::VALUE_OPTIONAL, 'ES query', null)
```

Comment:
```suggestion
            ->addOption(self::OPTION_QUERY, null, InputOption::VALUE_OPTIONAL, 'ES query', '{}')
```
I suggest to always do a queryin order to simplify the code